README.md

```diff
@@ -2,6 +2,18 @@
 
 This repository allows you to install [Apache Airflow](https://airflow.apache.org/) as a parcel deployable by [Cloudera Manager](https://www.cloudera.com/products/product-components/cloudera-manager.html).
 
+## Requirements
+- A supported operating system.
+- MySQL or PostgreSQL database in which to store Airflow metadata.
+
+### Currently Supported Versions of Airflow
+- Airflow 1.10
```

Comment:
versions available:
1.10
1.10.1
1.10.3

Reply:
Do we want to update the README every time we build? I left out patch-level versions just for that purpose.

Reply:
Ok that should be fine